plugins/services/src/js/components/dsl/TaskRegionDSLSection.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from "react";
+
+import DSLCombinerTypes from "#SRC/js/constants/DSLCombinerTypes";
+import DSLExpression from "#SRC/js/structs/DSLExpression";
+import DSLExpressionPart from "#SRC/js/structs/DSLExpressionPart";
+import DSLFormWithExpressionUpdates
+  from "#SRC/js/components/DSLFormWithExpressionUpdates";
+import DSLUtil from "#SRC/js/utils/DSLUtil";
+import FieldInput from "#SRC/js/components/form/FieldInput";
+import FieldLabel from "#SRC/js/components/form/FieldLabel";
+import FormGroup from "#SRC/js/components/form/FormGroup";
+
+const EXPRESSION_PARTS = {};
+
+class TaskRegionDSLSection extends React.Component {
+  render() {
+    const { expression, onChange } = this.props;
+    const { regions } = this.props.defaultData || [];
+
+    if (regions) {
```

Comment:
related to my comment above https://github.com/dcos/dcos-ui/pull/2472/files#r157902372